cmd/performance-profile-creator/cmd/root.go

```diff
@@ -94,7 +96,12 @@ func NewRootCommand() *cobra.Command {
 				return fmt.Errorf("failed to parse the info flag: %v", err)
 			}
 			if info {
-				showClusterData(cluster)
+				// intentionally undocumented and unsupported output to make e2e testing easier.
+				if _, ok := os.LookupEnv("PERFORMANCE_PROFILE_CREATOR_OUTPUT_JSON"); ok {
```

Comment:
The flow of showClusterDataJSON and showClusterDataLog is exactly the same other than one  stores in a data structure and the other print to logs. Maybe we can streamline this by only having a single showClusterData with isOutputJsonEnabled as an arg and only store in the data structure if  isOutputJsonEnabled is true. WDYT?

Reply:
sure, let's see how it looks.